libs/libcommon/src/libcommon/constants.py

```diff
@@ -48,20 +48,15 @@
 
 # Arrays are not immutable, we have to take care of not modifying them
 # Anyway: in all this file, we allow constant reassignment (no use of Final)
-CONFIG_HAS_VIEWER_KINDS = ["config-size"]
-CONFIG_INFO_KINDS = ["config-info"]
-CONFIG_PARQUET_METADATA_KINDS = ["config-parquet-metadata"]
-CONFIG_PARQUET_AND_METADATA_KINDS = ["config-parquet", "config-parquet-metadata"]
-CONFIG_SPLIT_NAMES_KINDS = ["config-split-names-from-info", "config-split-names-from-streaming"]
-DATASET_CONFIG_NAMES_KINDS = ["dataset-config-names"]
-DATASET_INFO_KINDS = ["dataset-info"]
-SPLIT_DUCKDB_INDEX_KINDS = ["split-duckdb-index"]
-SPLIT_HAS_PREVIEW_KINDS = ["split-first-rows-from-streaming", "split-first-rows-from-parquet"]
-SPLIT_HAS_SEARCH_KINDS = ["split-duckdb-index"]
-PARALLEL_STEPS_LISTS = [
-    CONFIG_SPLIT_NAMES_KINDS,
-    SPLIT_HAS_PREVIEW_KINDS,
-]
+CONFIG_HAS_VIEWER_KIND = "config-size"
+CONFIG_INFO_KIND = "config-info"
+CONFIG_PARQUET_METADATA_KIND = "config-parquet-metadata"
+CONFIG_SPLIT_NAMES_KIND = "config-split-names"
+DATASET_CONFIG_NAMES_KIND = "dataset-config-names"
+DATASET_INFO_KIND = "dataset-info"
+SPLIT_DUCKDB_INDEX_KIND = "split-duckdb-index"
+SPLIT_HAS_PREVIEW_KIND = "split-first-rows"
+SPLIT_DUCKDB_INDEX = "split-duckdb-index"
```

Comment:
Why not to use SPLIT_DUCKDB_INDEX_KIND instead?

Reply:
indeed. fixed